media/js/firefox/whatsnew/whatsnew-115-na-win7.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+// Create namespace
+if (typeof window.Mozilla === 'undefined') {
+    window.Mozilla = {};
+}
+
+(function () {
+    'use strict';
+
+    let url = window.location.href;
+    let hash;
+    let redirect;
+    let urlParts;
+
+    // strip hash from URL (if present)
+    if (url.indexOf('#') > -1) {
+        urlParts = url.split('#');
+        url = urlParts[0];
+        hash = urlParts[1];
+    }
+
+    if (url.indexOf('xv=windows') > -1) {
+        return;
+    } else {
+        redirect = url + (url.indexOf('?') > -1 ? '&' : '?') + 'xv=windows';
+
+        // re-insert hash (if originally present)
+        if (hash) {
+            redirect += '#' + hash;
+        }
+    }
+
+    if (
+        window.site.platform === 'windows' &&
+        window.site.platformVersion <= 6.1
+    ) {
```

Comment:
Perhaps add a check that `redirect` is defined here before redirecting, just to be safe?
```suggestion
    if (
        redirect &&
        window.site.platform === 'windows' &&
        window.site.platformVersion <= 6.1
    ) {
```